src/custom/utils/trade.ts

```diff
@@ -147,3 +150,9 @@ export async function sendOrderCancellation(params: OrderCancellationParams): Pr
 
   cancelPendingOrder({ chainId, id: orderId })
 }
+
+export async function hasTrades(chainId: ChainId, address: string): Promise<boolean> {
+  const trades = await getTrades(chainId, address)
```

Comment:
I know there's no pagination in the endpoint yet, however we know there will be
Why not making `getTrades` receive some params with `chainId`, `address` and an optional `limit` and `offset`.

Then, you would use `limit = 1` here.
Also, you would include the limit and offset params in the endpoint. It won't take effect for now, but nice to have it already since we know the "signature" it will have.

Reply:
Ah, that's why the change. Good having it now, doesn't hurt.